pennylane/transforms/qcut.py

```diff
@@ -1860,6 +2005,20 @@ def circuit(x):
             ) from e
 
     g = tape_to_graph(tape)
+
+    if auto_cutter is True or callable(auto_cutter):
+
+        cut_strategy = kwargs.pop("cut_strategy", None) or CutStrategy(
+            max_free_wires=len(device_wires)
+        )
+
+        g = find_and_place_cuts(
+            graph=g,
+            cut_method=auto_cutter if callable(auto_cutter) else kahypar_cut,
+            cut_strategy=cut_strategy,
+            **kwargs,
```

Comment:
Could we set `local_measurement=True` for `cut_circuit()`?

Reply:
Good catch! `cut_circuit()` could take it, but the `.expand_fn` couldn't. So I added in the signatures of the expansion functions `**kwargs` to ignore such arguments meant for cut method.